llvm-general/src/LLVM/General/Internal/OrcJIT/IRCompileLayer.hs

```diff
@@ -0,0 +1,76 @@
+{-# LANGUAGE MultiParamTypeClasses #-}
+module LLVM.General.Internal.OrcJIT.IRCompileLayer where
+
+import           LLVM.General.Prelude
```

Comment:
Please don't add whitespace after import like this.
In general, don't add whitespace just to line things up, and attempt to style stuff as in the prexisting code.